### PR TITLE
fix: Action - Remove padding when used without an icon

### PR DESCRIPTION
**Pull request by driskull**
Fix display for text without icon. Previously, there was some padding to the left of the text for no reason.

**Related Issue:** None

## Summary

@asangma I noticed that the use of an action without an icon produced padding unnecessarily.

image

### CODE REVIEW

File: src/components/calcite-action/calcite-action.tsx [hunk 1]
```diff
@@ -119,6 +119,8 @@ export class CalciteAction {
   renderIconContainer(): VNode {
     const { loading } = this;
 
+    const hasIcon = !!this.el.children.length;
```
Comment (jcfranco) [inline src/components/calcite-action/calcite-action.tsx]: Did Stencil fix this recently? I remember I had an issue with similar logic in Edge.

From https://github.com/Esri/calcite-app-components/pull/785:

> This restores block toggling in Edge. The issue was caused by the 'has content' logic, which caused the block to never open after being closed (`el.children` in Edge includes the elements in the render function, tripping up the logic).

Comment (driskull) [reply src/components/calcite-action/calcite-action.tsx]: dangit. Do you know a better way to figure out if the element has content?

Comment (jcfranco) [reply src/components/calcite-action/calcite-action.tsx]: Not at the moment. 😢

**Comment by driskull** [comment]
cc @kevindoshier. Noticed this on mapviewer

**Pull request merged**